batch/batch/cloud/gcp/worker/disk.py

```diff
@@ -57,12 +60,17 @@ async def _unmount(self):
 
     async def _format(self):
         async def format_disk():
-            await check_shell_output(
-                f'mkfs.ext4 -m 0 -E lazy_itable_init=0,lazy_journal_init=0,discard {self.disk_path}'
-            )
-            await check_shell_output(f'mkdir -p {self.mount_path}')
-            await check_shell_output(f'mount -o discard,defaults {self.disk_path} {self.mount_path}')
-            await check_shell_output(f'chmod a+w {self.mount_path}')
+            try:
+                await check_shell_output(
+                    f'mkfs.ext4 -m 0 -E lazy_itable_init=0,lazy_journal_init=0,discard {self.disk_path}'
+                )
+                await check_shell_output(f'mkdir -p {self.mount_path}')
+                await check_shell_output(f'mount -o discard,defaults {self.disk_path} {self.mount_path}')
+                await check_shell_output(f'chmod a+w {self.mount_path}')
+            except CalledProcessError:
+                outerr = await check_shell_output(f'ls {os.path.dirname(self.disk_path)}')
+                log.info(f'debugging info while formatting disk for {self.name}: {outerr}\n{self.last_response}')
```

Comment:
Can we capture the exception info that got us here? Also what if the `ls` errors?